internal/engine/actions.go

```diff
@@ -126,10 +121,17 @@ func NewHudStoppedAction(err error) HudStoppedAction {
 	return HudStoppedAction{err}
 }
 
-type TiltfileReloadedAction struct {
-	Manifests  []model.Manifest
-	GlobalYAML model.YAMLManifest
-	Err        error
+type ConfigsReloadStartedAction struct {
+	FilesChanged map[string]bool
```

Comment:
It's slightly more memory efficient and idiomatic to represent maps as `map[X]struct{}`.

Reply:
More on this [here](https://dave.cheney.net/2014/03/25/the-empty-struct).

Reply:
FWIW, I was on this side at my last gig, and lost: https://golang.org/doc/effective_go.html#maps

"A set can be implemented as a map with value type bool."

Reply:
Oh yeah that makes sense. I take it back!

Reply:
huh okay! but fwiw this is mirroring `EngineState.PendingConfigFileChanges` which is a `map[string]bool`. I'll fix mon if i have time.